server/interfaces/api/userSettingsInterfaces.ts

```diff
@@ -12,14 +15,16 @@ export interface UserSettingsGeneralResponse {
   globalTvQuotaDays?: number;
 }
 
+export type NotificationAgentTypes = Record<NotificationAgentKey, number>;
 export interface UserSettingsNotificationsResponse {
-  notificationAgents: number;
   emailEnabled?: boolean;
+  webPushEnabled?: boolean;
   pgpKey?: string;
   discordEnabled?: boolean;
   discordId?: string;
   telegramEnabled?: boolean;
   telegramBotUsername?: string;
   telegramChatId?: string;
   telegramSendSilently?: boolean;
```

Comment:
```suggestion
  emailEnabled?: boolean;
  pgpKey?: string;
  discordEnabled?: boolean;
  discordId?: string;
  telegramEnabled?: boolean;
  telegramBotUsername?: string;
  telegramChatId?: string;
  telegramSendSilently?: boolean;
  webPushEnabled?: boolean;
```
Can we keep related fields grouped together here? 🐱

Reply:
(And in `server/routes/user/usersettings.ts` as well 🙏🏻)